packages/docs/src/components/doc/ApplySponsor.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div>
+    <p>
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptatem
```

Comment:
Not sure you mean to keep lorem ipsum here

Reply:
No, will change it